app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Page.php

```diff
@@ -18,28 +20,47 @@
 class Page
 {
     /**
-     * @var FilterEmulate
+     * @var GetPageByIdentifierInterface
      */
-    private $widgetFilter;
+    private $pageByIdentifier;
 
     /**
      * @var PageRepositoryInterface
      */
     private $pageRepository;
 
     /**
-     * @param PageRepositoryInterface $pageRepository
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * @var FilterEmulate
+     */
+    private $widgetFilter;
+
+    /**
+     * @param GetPageByIdentifierInterface $getPageByIdentifier
      * @param FilterEmulate $widgetFilter
+     * @param PageRepositoryInterface $pageRepository
+     * @param StoreManagerInterface $storeManager
      */
     public function __construct(
+        GetPageByIdentifierInterface $getPageByIdentifier,
```

Comment:
1. \Magento\CmsGraphQl\Model\Resolver\DataProvider\Page::__construct
You have changed the order of original arguments; That is not okay according to backwards-compatibility policy.
2. \Magento\CmsGraphQl\Model\Resolver\DataProvider\Page::__construct
Check "Adding a constructor parameter" section here https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/
That was done not according to backwards-compatibility policy.

Reply:
**done**, sir